worker_binder.go

```
@@ -0,0 +1,93 @@
+
+	return newQueue[T, R](qs.worker)
+}
+
+func (q *workerBinder[T, R]) BindPriorityQueue() PriorityQueue[T, R] {
+	defer q.worker.start()
+
+	return newPriorityQueue[T, R](q.worker)
+}
+
+func (q *workerBinder[T, R]) BindWithPersistentQueue(pq IQueue) PersistentQueue[T, R] {
+	defer q.worker.start()
```

golangci-lint (1.64.8)

Error return value of `q.worker.start` is not checked

(errcheck)